fixture-monkey-api/src/main/java/com/navercorp/fixturemonkey/api/matcher/MatcherOperatorRegistry.java

```diff
@@ -0,0 +1,11 @@
+package com.navercorp.fixturemonkey.api.matcher;
+
+import java.util.List;
+
+import com.navercorp.fixturemonkey.api.property.Property;
+
+public interface MatcherOperatorRegistry<T> {
```

Comment:
Please add `@API(since = "1.1.17", status = EXPERIMENTAL)`

Reply:
The change was applied, but it does not appear in the git diff.